templates/repo/clone_panel.tmpl

```diff
@@ -0,0 +1,38 @@
+<div class="menu gt-mt-3 clone dropdown">
```

Comment:
Please use `.clone-dropdown` instead of `.clone.dropdown` for this and further similar cases. The former is much more resistant to accidential class name collision.

Reply:
I'll update these class names

Reply:
It shouldn't be called as `dropdown` anymore IMO. 

There is a `ui dropdown` from Fomantic UI outside. So it's better not using the conflicting name for our components.

Reply:
What name would you suggest instead? Something like `clone-panel-content`?

Reply:
Yup, it could be clearer.

Reply:
Done